package.json

```diff
@@ -2,11 +2,16 @@
   "name": "controlcenter",
   "version": "0.0.0",
   "dependencies": {
-    "react": "^0.12.2"
+    "es5-shim": "^4.1.0",
```

Comment:
If it is required by reflux, why is it added as a high-level dependency to the project itself?